src/dinifile.d

```diff
@@ -72,7 +72,7 @@ const(char)* findConfFile(const(char)* argv0, const(char)* inifile)
     filename = FileName.replaceName(argv0, inifile);
     if (FileName.exists(filename))
         return filename;
-    static if (__linux__ || __APPLE__ || __FreeBSD__ || __OpenBSD__ || __sun)
+    static if (HOST_LINUX || HOST_OSX || HOST_FREEBSD || HOST_OPENBSD || HOST_SOLARIS)
```

Comment:
You're missing the point.  I couldn't care less about removing uses of static if, I just want to rename these identifiers so they don't conflict with C++ preprocessor builtins.